src/DataTables/assets/src/controller.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import { Controller } from '@hotwired/stimulus';
+import DataTable from 'datatables.net-dt';
+
+let isDataTableInitialized = false;
+
+export default class extends Controller {
+    declare readonly viewValue: any;
+
+    static values = {
+        view: Object,
+    };
+
+    private table: DataTable | null = null;
+
+    connect() {
+        if (isDataTableInitialized) {
```

Comment:
This will prevent using multiple datatables on a single page. This can be a private prop on the controller